src/operators.jl

```diff
@@ -398,15 +418,45 @@ function _multiply!{T<:Union{GenericAffExpr,GenericQuadExpr}}(ret::Array{T}, lhs
     ret
 end
 
+# this computes lhs.'*rhs and places it in ret
+function _multiplyt!{T<:Union{GenericAffExpr,GenericQuadExpr}}(ret::Array{T}, lhs::Matrix, rhs::SparseMatrixCSC)
+    rowval = rhs.rowval
+    nzval  = rhs.nzval
+    for multivec_row ∈ 1:size(lhs,2) # transpose
+        for col ∈ 1:rhs.n
+            idxset = rhs.colptr[col]:(rhs.colptr[col+1]-1)
```

Comment:
If you could update ``_multiply!`` above for consistency then I think we're all set